pages/editor.vue

```diff
@@ -313,19 +315,10 @@
 
     .darken-background {
         width: 100vw;
-        height: 100vh;
-        background-color: $black;
-        opacity: 60%;
-    }
-
-    .comic-preview {
-        width: 60vw;
-        height: 20vh;
-        background-color: $grey-70;
-        position: absolute;
-        top: 50%;
-        left: 50%;
-        transform: translate(-50%, -50%);
+        height: 100%;
+        background-color: $grey-60;
```

Comment:
```suggestion
        background-color: rgba($grey-60, 0.4);
```

Reply:
this is not enough, but I managed to make it work :)